migrations/sqls/20201028120146-billing-invoice-licences-constraint-changes-up.sql

```diff
@@ -1,6 +1,7 @@
-ALTER TABLE water.billing_invoice_licences ADD CONSTRAINT billing_invoice_licences_unique UNIQUE (billing_invoice_id,licence_id);
+ALTER TABLE water.billing_invoice_licences DROP CONSTRAINT IF EXISTS unique_batch_invoice_licence;
 ALTER TABLE water.billing_invoice_licences DROP COLUMN company_id;
 ALTER TABLE water.billing_invoice_licences DROP COLUMN contact_id;
 ALTER TABLE water.billing_invoice_licences DROP COLUMN address_id;
 ALTER TABLE water.billing_invoice_licences DROP COLUMN licence_holder_name;
 ALTER TABLE water.billing_invoice_licences DROP COLUMN licence_holder_address;
+ALTER TABLE water.billing_invoice_licences ADD CONSTRAINT billing_invoice_licences_unique UNIQUE (billing_invoice_id,licence_id);
```

Comment:
Is this related to the recent work on ensuring only 1 invoiceLicence per licence per invoice?

Reply:
This was an issue when I was moving up and down through migrations when billing data was present in my local database. Is it incorrect?

Reply:
I have tested locally and they seem to work fine for me too